### ISSUE TITLE
Frontier API invalid cast exception

### ISSUE THREAD

**Issue by Tkael** [EDCD/EDDI#1207]
## EDDI version in which issue found
3.3.6

## Steps to reproduce

1. Trigger a station update via the Frontier API
2. Observe the parser during the response

## Expected
No exceptions.

## Observed
```
System.InvalidCastException: Unable to cast object of type 'Newtonsoft.Json.Linq.JObject' to type 'Newtonsoft.Json.Linq.JProperty'. 
at EddiCompanionAppService.CompanionAppService.ShipyardFromProfile(JObject json) 
at EddiCompanionAppService.CompanionAppService.Station(String systemName) 
at Eddi.EDDI.conditionallyRefreshProfile()
```

## Investigation
[any investigation you have done, regression tests against earlier versions, etc]

**Comment by richardbuckle** [comment]
Ref https://rollbar.com/EDCD/EDDI/items/12556/occurrences/69914950925/

**Comment by richardbuckle** [comment]
Fixed in 3.3.76-b1
